text_formatter.go

```diff
@@ -102,7 +103,7 @@ func (f *TextFormatter) isColored() bool {
 		}
 	}
 
-	return isColored && !f.DisableColors
+	return isColored && !f.DisableColors && (runtime.GOOS != "windows")
```

Comment:
Hello, I would like that the ForceColor field of the text formatter takes precedence on the environment check.
I expect that then this flag is set to true the user knows what he is doing.